ur_controllers/CMakeLists.txt

```
@@ -1,35 +1,52 @@
 cmake_minimum_required(VERSION 3.5)
 project(ur_controllers)
 
-add_compile_options(-std=c++11)
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+    set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+    add_compile_options(-Wall -Wextra)
```

The old file used two spaces for indentation. let's keep this...
```suggestion
  add_compile_options(-Wall -Wextra)
```